### ISSUE TITLE
Support for macOS system Ruby: 2.6.8

### ISSUE THREAD

**Issue by iandol** [htdebeer/pandocomatic#107]
On macOS, the system version of Ruby is 2.6.8. Paru recently stopped working on this default version and you set the minimum version back to 2.6.8 here without any obvious problems: https://github.com/htdebeer/paru/issues/72 

pandocomatic has now stopped working on macOS:

```
▷ sudo gem update pandocomatic
Password:
Updating installed gems
Updating pandocomatic
Fetching pandocomatic-1.0.3.gem
ERROR:  Error installing pandocomatic:
	There are no versions of pandocomatic (= 1.0.3) compatible with your Ruby & RubyGems
	pandocomatic requires Ruby version >= 2.7.6. The current ruby version is 2.6.8.205.
```

The same fix in the gemfile would be welcome for `pandocomatic`. Thanks!

**Comment by htdebeer** [comment]
Right, I thought pandocomatic was still on an old version, but not old enough!

I've fixed this, it is in version 1.0.4.

**Comment by iandol** [comment]
Confirmed working, thanks as always Huub ❤️
